### PR TITLE
Droth 358 geometry with new attributes fetched from vvh

### PR DESCRIPTION
**Pull request by halied**

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala [hunk 1]
```diff
@@ -537,7 +544,7 @@ class VVHClient(vvhRestApiEndPoint: String) {
       "HORIZONTALACCURACY",
       "VERTICALACCURACY",
       "VERTICALLEVEL",
-      "CONSTRUCTIONTYPE",
+      "CONSTRUCTIONTYPE",//TODO Remove this attribure from here because now is a instance variable
```
Comment (halied) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala]: Is this TODO still needed?

File: UI/src/view/navigationpanel/ActionPanelBoxes.js [hunk 2]
```diff
@@ -188,6 +188,19 @@
         '  </div>'
     ].join('');
 
+    var historyLegend = [
```
Comment (halied) [inline UI/src/view/navigationpanel/ActionPanelBoxes.js]: Should this be constructionTypeLegend?

**Comment by halied** [comment]
OK

**Comment by leirasf** [comment]
Beside the Hanna commets it's ok to me

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala (hunk 1 above)
Comment (leirasf) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala]: I belive that now we don't need to have CONSTRUCTIONTYPE on attribute list.

File: UI/src/view/navigationpanel/ActionPanelBoxes.js (hunk 2 above)
Comment (marquesfm) [inline UI/src/view/navigationpanel/ActionPanelBoxes.js]: Modified to constructionTypeLegend.

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala (hunk 1 above)
Comment (marquesfm) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala]: It's better don't remove this attribute until the VVHHistoryRoadLink be modified like VVHRoadlink was.
We will modified the TODO for keep that information.

**Comment by marquesfm** [comment]
It's ok to me.

**Pull request merged**